content/zh-cn/docs/tasks/inject-data-application/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
 title: "给应用注入数据"
-weight: 30
+weight: 70
```

Comment:
The english frontmatter can be copied and commented here, to keep consistent with other pages.